templates/admin.php

```diff
@@ -65,4 +65,31 @@
 
 	<input type="checkbox" id="enable_menu_option_cb-richdocuments" <?php p($_['menu_option'] !== 'false' ? 'checked' : '') ?> />
 	<label for="enable_menu_option_cb-richdocuments"><?php p($l->t('Use Menu option')) ?></label>
+
+	<br/>
+	<input type="checkbox" id="enable_secure_view_option_cb-richdocuments" <?php if ($_['secure_view_option'] === 'true') {
+	p('checked />');
+} elseif ($_['secure_view_allowed'] !== 'true') {
+	p('disabled />');
+} else {
+	p(' />');
+}?>
+	<label for="enable_secure_view_option_cb-richdocuments"><?php p($l->t('Enable Secure View for read-only shares (requires enterprise version)')) ?></label>
+	<div id="enable-share-attributes-defaults" style="padding-left: 28px;" class="indent <?php if ($_['secure_view_option'] !== 'true') {
+	p('hidden');
+} ?>" >
+		<p style="max-width: 50em;"><em><?php p($l->t('Set default share permissions for all users')) ?></em></p>
+		<input type="checkbox" id="secure_view_can_download_default_option_cb-richdocuments" <?php p($_['secure_view_can_download_default'] === 'true' ? 'checked' : '') ?> />
+		<label for="secure_view_can_download_default_option_cb-richdocuments"><?php p($l->t('can view with watermark (disabling allows only viewing with watermark)')) ?></label>
```

Comment:
please use the same text as for the checkboxes in the share dialog

cc @pmaier1